packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx

```diff
@@ -140,9 +141,9 @@ export const CoinjoinConfirmation = ({ account }: CoinjoinConfirmationProps) =>
         await dispatch(
             startCoinjoinSession(account, {
                 maxCoordinatorFeeRate: coinjoinClient.coordinationFeeRate.rate,
-                maxFeePerKvbyte: coinjoinClient.maxMiningFee * 1000, // transform to kvB
+                maxFeePerKvbyte: coinjoinAccount?.maxFeePerKvbyte * 1000, // transform to kvB
```

Comment:
`coinjoinClient.maxMiningFee` should be still the default value, right? Trezor wants me to confirm 3 sat TEST/vB even though coinjoin.clients['test'].maxMiningFee is 5 - client value should be propagated to account value

Reply:
having both mainnet and testnet activated, custom setup on mainnet, default on testnet and trying to start coinjoin on testnet account
```json
{
    "accounts": [
        {
            "key": "...",
            "symbol": "btc",
            "rawLiquidityClue": null,
            "targetAnonymity": 5,
            "maxFeePerKvbyte": 115,
            "skipRounds": [
                4,
                5
            ],
            "previousSessions": [],
            "checkpoints": [...],
            "customSetup": true
        },
        {
            "key": "...",
            "symbol": "test",
            "rawLiquidityClue": 118098,
            "targetAnonymity": 5,
            "maxFeePerKvbyte": 3,
            "previousSessions": [],
            "checkpoints": [...],
    "clients": {
        "btc": {
            "coordinationFeeRate": {
                "plebsDontPayThreshold": 1000000,
                "rate": 0.003
            },
            "maxMiningFee": 14,
            "allowedInputAmounts": {
                "max": 134375000000,
                "min": 5000
            },
            "rounds": [
                {
                    "id": "495aeeedc3bb612f126dfd64d9e06865a842e53dc34f1a03b43a71f279867975",
                    "phase": 4
                },
                {
                    "id": "711948a22c3c4141a746c45a88397bceb30832cba45db3166200d993bf733c95",
                    "phase": 0
                },
                {
                    "id": "ae6d736aabe8185c6a8baa33c55ed67c94054e517c5972d3e4da7329ae8c45db",
                    "phase": 4
                }
            ],
            "status": "loaded"
        },
        "test": {
            "coordinationFeeRate": {
                "plebsDontPayThreshold": 1000000,
                "rate": 0.003
            },
            "maxMiningFee": 5,
            "allowedInputAmounts": {
                "max": 134375000000,
                "min": 5000
            },
            "rounds": [
                {
                    "id": "305c4b53cb7d6dd07b6a214906236d8ccaffdf6bbfcd5b938d44ecd4d3b1ccaa",
                    "phase": 4
                },
                {
                    "id": "f57ec832d482a60543573475f71dbcf6c4536ff89004989acf13a4a9cdb1d488",
                    "phase": 4
                },
                {
                    "id": "72a3d9795804a26a3b7a8b2d5568a9954e7ba5dab42c7f92f1687b422a14775e",
                    "phase": 0
                },
                {
                    "id": "4dc6ab82c22df3cce19a057605aacc576728fde633c296fad55b15496ffdac2b",
                    "phase": 4
                },
                {
                    "id": "aa30dbc7c8c4663f092f99135df8055318d29488d977a3eb4b44974daae97114",
                    "phase": 0
                }
            ],
            "status": "loaded"
        }
    },
    "isPreloading": false,
    "config": {
        "averageAnonymityGainPerRound": 5,
        "roundsFailRateBuffer": 10,
        "roundsDurationInHours": 1
    }
}
```

Reply:
I see... it remembered the `maxFeePerKvbyte` value from the account's creation. Fixed by defaulting to `client.maxMiningFee` in f0e82251d4edf0828f2b757b558c3c50e46b8f79. I think I should add one more commit to treat `targetAnonymity` and `skipRounds` the same way, i.e. only have them in the reducer when custom setup is on, otherwise just use the default value. As it is currently implemented, we cannot e.g. increase `targetAnonymity` of existing accounts without a migration. Perhaps also change the data structure to:
```ts
setup?: {
    targetAnonymity: number,
    maxFeePerKvbyte: number,
    skipRounds: boolean
}
```

Reply:
There it is: f1480faa86cf0e822dc949b1f50d8db652c2746d